<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>startForm</class>
 <widget class="QDialog" name="startForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>224</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>416</width>
    <height>224</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>416</width>
    <height>224</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Inspect - LegUp HLS Source-Level Debugger 1.0</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Debugging.png</normaloff>Debugging.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>372</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Please select one of the below modes and click on Start:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStart">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>31</y>
     <width>241</width>
     <height>164</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radioButtonSimulationRun">
      <property name="text">
       <string>Simulation Run using ModelSim</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonOnChipRun">
      <property name="text">
       <string>OnChip Run using Quartus II</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonGDB">
      <property name="text">
       <string>GDB Run</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonGDBSync">
      <property name="text">
       <string>GDB Sync Mode</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonBugDetection">
      <property name="text">
       <string>Bug Detection Mode</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonOnChipBugDetection">
      <property name="text">
       <string>OnChip Bug Detection</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonStart</sender>
   <signal>clicked()</signal>
   <receiver>startForm</receiver>
   <slot>pushButtonStart_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startForm</sender>
   <signal>rejected()</signal>
   <receiver>startForm</receiver>
   <slot>startForm_rejected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pushButtonStart_clicked()</slot>
  <slot>startForm_rejected()</slot>
 </slots>
</ui>
